crates/bitwarden-crypto/src/keys/shareable_key.rs

```diff
@@ -20,14 +20,15 @@ pub fn derive_shareable_key(
     // TODO: Are these the final `key` and `info` parameters or should we change them? I followed
     // the pattern used for sends
     let res = Hmac::<sha2::Sha256>::new_from_slice(format!("bitwarden-{}", name).as_bytes())
```

Comment:
Mind changing this to be `PbkdfSha256Hmac::new_from_slice` while at it?

Reply:
Done